tests/seta/test_models.py

```diff
@@ -0,0 +1,90 @@
+import datetime
+
+import pytest
+from django.db.utils import IntegrityError
+from django.utils import timezone
+
+from treeherder.seta.models import (JobPriority,
+                                    TaskRequest)
+
+TOMORROW = timezone.now() + datetime.timedelta(days=1)
+YESTERDAY = timezone.now() - datetime.timedelta(days=1)
+slow = pytest.mark.slow
+
+
+# TaskRequest tests
+@slow
+def test_create_instance(test_repository):
+    TaskRequest.objects.create(repository=test_repository,
+                               counter=0,
+                               last_request=timezone.now(),
+                               reset_delta=0)
+
+
+# JobPriority tests
+def test_expired_job_priority():
+    jp = JobPriority(testtype='web-platform-tests-1',
+                     buildtype='opt',
+                     platform='windows8-64',
+                     priority=1,
+                     timeout=5400,
+                     expiration_date=YESTERDAY,
+                     buildsystem='taskcluster')
+    assert jp.has_expired() is True
```

Comment:
I would like to make sure that the "has_expired" method works as expected.

It's not much code and runs fast. Do you want me to remove them? or do it differently?